### ISSUE TITLE
Setting OAuth tokens timedelta throws Error 

### ISSUE THREAD

**Issue by Kaleidophon** [stormpath/stormpath-sdk-python#290]
Hi!

I am trying to use Stormpath in my API built with Python and Flask(-RESTful).

When I try to set up the API, Stormpath will throw an error while setting the expiration of access and refresh tokens with values read from a config file:
```python
Traceback (most recent call last):
  File "./app.py", line 13, in <module>
    app = setup_app(config_path="config.json")
  File "./runserver.py", line 47, in setup_app
    OAuthPolicyResource.prepare(app)
  File "./resources/auth_resources.py", line 189, in prepare
    setattr(app.oauth_policy, "access_token_ttl", cls.access_token_ttl)
  File "/usr/local/lib/python2.7/site-packages/stormpath/resources/base.py", line 129, in __getattr__
    self._ensure_data()
  File "/usr/local/lib/python2.7/site-packages/stormpath/resources/base.py", line 272, in _ensure_data
    data = self._store.get_resource(self.href, params=params)
  File "/usr/local/lib/python2.7/site-packages/stormpath/data_store.py", line 188, in get_resource
    data = self.executor.get(href, params=params)
  File "/usr/local/lib/python2.7/site-packages/stormpath/http.py", line 174, in get
    return self.request('GET', url, params=params)
  File "/usr/local/lib/python2.7/site-packages/stormpath/http.py", line 160, in request
    raise Error({'developerMessage': message})
stormpath.error.Error: Trying to redirect outside of API base url: https://api.stormpath.com/login
```

Oddly enough, this only happens when the API is running on one of our company servers, but not on my local machine. What could this be due to?

Thanks!

**Comment by rdegges** [comment]
Sounds to me like your company servers have some sort of web proxy which is forcibly stealing your HTTP connections. Can you check to see if you have any corporate firewalls / proxies and if so, what they do?

**Comment by Kaleidophon** [comment]
After digging into your library's code, I found out that the Exception is thrown because a criterion in `http.py` isn't met, where a response from the server is supposed to contain an http-code of `200`, but contains `301` or `302`.
We looked into the request the Stormpath library sends out during this step on my local machine to the SP API and sent the same one manually via `curl` and python's `request` library from the docker container on our server that the app is running in. Both of those worked correctly , but somehow the same request doesn't seem to work sent from inside the SP library.

We noticed that during the requests, an authorization token is also sent as a header. This token seems to be generated dynamically during a session (?), so we figured that the problem might be due to this (we copied this token from the request sent from the local machine and used it on the server), but unfortunately there is no way for us to debug this and find out effectively.
